app/assets/javascripts/discourse/app/routes/review-index.js

```diff
@@ -83,10 +79,17 @@ export default DiscourseRoute.extend({
 
   deactivate() {
     this.messageBus.unsubscribe("/reviewable_claimed");
+    this.messageBus.unsubscribe(this._reviewableCountsChannel());
```

Comment:
What is the context on why we need to unsubscribe this? 

This unsubscribe undo the subscription that is set up here: 

https://github.com/discourse/discourse/blob/51cabf0f26e66e859e6a9bb898ad3507ae1d0c28/app/assets/javascripts/discourse/app/initializers/subscribe-user-notifications.js#L28-L40

Reply:
I actually remove the subscription in https://github.com/discourse/discourse/pull/18374#discussion_r980778591

Reply:
> This unsubscribe undoes the subscription that is set up here:

Oh, yeah, all message bus unsubs should target a specific function, not the whole channel

Reply:
ahh I forgot that `unsubscribe` takes in a function as well: 

https://github.com/discourse/message_bus/blob/f67c762c717ecfc1d908ef75491236c048921a99/assets/message-bus.js#L509

cc @OsamaSayegh

Reply:
I think we definitely need to unsubscribe otherwise a callback would leak everytime you enter and leave the review route, but yeah it should unsubscribe a specific function.

Reply:
Ah good point, yea the correct solution should have been to unsubscribe a specific function and not nuke it xD